src/nrnpython/CMakeLists.txt

```diff
@@ -24,14 +24,17 @@ if(NRN_ENABLE_PYTHON_DYNAMIC)
       ../nrnoc
       ../ivoc
       ../nrniv
-      ../ivos
       ../gnu
       ../mesch
       ../nrnmpi
       ${PROJECT_BINARY_DIR}/src/nrnpython
       ${PROJECT_BINARY_DIR}/src/ivos
-      ${PROJECT_BINARY_DIR}/src/oc
-      ${IV_INCLUDE_DIR})
+      ${PROJECT_BINARY_DIR}/src/oc)
+  if(NRN_ENABLE_INTERVIEWS)
+    list(APPEND INCLUDE_DIRS ${IV_INCLUDE_DIR})
+  else()
+    list(APPEND INCLUDE_DIRS ../ivos)
+  endif()
```

Comment:
I can't review this. @pramodk might know.

Reply:
I think this is correct, @nrnhines do you agree?

Reply:
looks like ivos is present twice. lines 31 and 36 . But after a master build,install using
```
-DNRN_ENABLE_INTERVIEWS=OFF -DNRN_ENABLE_PYTHON_DYNAMIC=ON
```
 I  see that in the build folder, ```src/ivos``` does not exist.

Reply:
I don't see include files generated by an interviews build except
```
~/neuron/nrn/build$ find . -name config.h
./external/iv/config.h
```
But I don't think that is needed  for anything except the build portion of interviews.

Reply:
So I will remove this include in a following PR